src/server/req_manager.c

```diff
@@ -1627,6 +1627,9 @@ mgr_server_unset(struct batch_request *preq)
 			if (dflt_scheduler) {
 				svrattrl *tm_list;
 				tm_list = attrlist_create(plist->al_name, NULL, 0);
+				if (tm_list == (svrattrl *)0) {
```

Comment:
I know the server uses (cast)0, but I'd sure rather see you use NULL instead.  What do you think?

Reply:
Implemented.